packages/utils/tailwind-plugins/package.json

```diff
@@ -7,7 +7,8 @@
   },
   "main": "index.js",
   "dependencies": {
-    "@spark-ui/theme-utils": "^2.10.0"
+    "@spark-ui/theme-utils": "^2.9.1",
+    "tailwindcss-radix": "^2.8.0"
```

Comment:
"@spark-ui/theme-utils": "2",
 "tailwindcss-radix": "2.8.0"

fix the versions